recipe/meta.yaml

```diff
@@ -1,6 +1,5 @@
 {% set name = "mlflow" %}
-{% set version = "2.9.2" %}
-{% set sha256 = "25f2ad3d0b0e78bd32296058fb4c315dff4379515cd669ac6137f39507b9dfe9" %}
+{% set version = "2.12.1" %}
```

Comment:
There is a new release `2.12.2` that fixes several bugs https://github.com/mlflow/mlflow/blob/master/CHANGELOG.md#2122-2024-05-08. The difference of depenency's pinnings is only protobuf and gunicorn https://github.com/mlflow/mlflow/compare/v2.12.1...v2.12.2

Reply:
Done!